specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/examples/GetAllTestFiles.json

```diff
@@ -1,7 +1,8 @@
 {
   "parameters": {
     "testId": "12345678-1234-1234-1234-123456789012",
-    "api-version": "2022-06-01-preview"
+    "api-version": "2022-06-01-preview",
+    "continuationToken": "continuation token"
```

Comment:
Can we put the same description that we have

Reply:
Which description?

Reply:
We have added description in the model right, so in the examples continuation token would be the value right?

Reply:
Ok I guess this is example